scripts/generate-changelog.js

```diff
@@ -0,0 +1,90 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const readline = require('readline')
+const replace = require('replace-in-file')
+const util = require('util')
+const exec = util.promisify(require('child_process').exec)
+const spawn = util.promisify(require('child_process').spawn)
+
+const name = require('emoji-name-map')
+
+const lernaConfig = require('../lerna.json')
+
+const CHANGELOG_FILE = 'CHANGELOG.md'
+
+async function main() {
+  const lastHashCmd = await exec('git rev-list --tags --max-count=1')
```

Comment:
```suggestion
  const lastHashCmd = await exec('git rev-parse HEAD')
```
It seems a bit more simple to me

Reply:
What I'm trying to get is the commit hash of the last previous tag not HEAD's commit hash.

Reply:
oh ok, so some renaming could make that clearer, what about:
- lastHashCmd -> lastTagHashCmd
- oldTagCmd -> lastTagNameCmd